user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_reinitialize_resourcemanager.md

```diff
@@ -0,0 +1,19 @@
+---
+uid: SLNetClientTest_reinitialize_resourcemanager
+---
+
+# Reinitializing ResourceManager
+
+From DataMiner 10.3.9/10.4.0 onwards, you can (re)initialize ResourceManager using the SLNet Client Test Tool.<!-- RN36811 -->
+This can be useful if ResourceManager failed to initialize on DataMiner startup, and you want to retry initializing without restarting DataMiner.
+
+> [!NOTE]
+> You require the *'Resources > Edit > Other'* permission flag in order to do this.
```

Comment:
Just leaving a comment here so we don't forget the change this in case we change the necessary permission flag.